krun/vm_defs.py

```diff
@@ -480,12 +579,105 @@ def __init__(self, vm_path, env=None):
         lib_dir = os.path.dirname(vm_path)
         self.add_env_change(EnvChangeAppend("LD_LIBRARY_PATH", lib_dir))
 
+    def parse_instrumentation_stderr_file(self, file_handle):
+        """PyPy instrumentation data collected from the PYPYLOG.
+
+        We collect:
+          * The time spent in compilation-related tasks.
+          * The time spent in GC-related tasks.
+
+        Note that the times are not in wall-clock time. Consider the units
+        arbitrary.
```

Comment:
FWIW, the time is in terms of CPU high performance counter. It might be clock cycles? Anyway, I doubt that it's possible to turn it into something useful non-relative.